R/utils-json.R

```diff
@@ -218,12 +218,14 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, po
     working_index <- getApprovalIndex(data, points, chain_nm, "Working");
     review_index <- getApprovalIndex(data, points, chain_nm, "In Review");
     approved_index <- getApprovalIndex(data, points, chain_nm, "Approved");
+    admin_index <- getApprovalIndex(data, points, chain_nm, "Admin Read-Only");
     
     review_index <- setdiff(review_index, working_index)
     approved_index <- setdiff(approved_index, working_index)
     approved_index <- setdiff(approved_index, review_index)
+    approved_index <- setdiff(approved_index, admin_index)
```

Comment:
The diffs work by removing any values from the first vector that also exist in the second vector. The logic here is to make the dv approval points show for the lowest approval status on any given day.

The three indices are initially filled with each day that this approval status is present on, and then in order to make the lowest approval status be the one that shows up for any given day we remove duplicates from the higher level status indices, leaving those days in the lower status indices. 

So in this case we:

1. Remove any points from "In Review" that are also in "Working" because "Working" is the lower status. 
2. Remove any points from "Approved" that are also in "In Review" because "In Review" is the lower status.
3. Remove any points from "Approved" that are also in "Working" because "Working" is the lower status.

Both "Working" and "In Reivew" have to be removed from "Approved" because there is not a guarantee that an "In Review" period exists between a "Working" period and an "Approved" period.

So in the case of adding this new status that is the top dog we would need all of the following because we want DV points to show the lowest approval status on any given day:

admin_index <- setdiff(admin_index, working_index)
admin_index <- setDiff(admin_index, review_index)
admin_index <- setDiff(admin_index, approved_index)